examples/faq.md

```diff
@@ -0,0 +1,36 @@
+### FAQ
+
+1. My data is not logged properly:
+
+- There could be chances that there are race condition where the main thread is exiting before the writer thread, hence you could avoid this using `.close()` method of `mlboard::filewriter` object
+
+2. Error : Detected out of order event.step:
+
+- If you get the following error 
+
+` Detected out of order event.step likely caused by a TensorFlow restart. Purging 1 expired tensor events from Tensorboard display between the previous step: 24 (timestamp: 1594962994.0) and current step: 1 (timestamp: 1594962994.0).`
+`
+
+There are chances that there are two event files in the `logdir`. Make sure that every logdir has only one event file. On every run create a new logdir either by use timestamp or hash function to avoid such situations.
```

Comment:
These lines are not comming properly indented I think checkout display rich diff option to see what I am trying to say.